src/pages/Swap/index.tsx

```diff
@@ -65,7 +65,8 @@ import { computeFiatValuePriceImpact } from '../../utils/computeFiatValuePriceIm
 import { maxAmountSpend } from '../../utils/maxAmountSpend'
 import { computeRealizedPriceImpact, warningSeverity } from '../../utils/prices'
 import { supportedChainId } from '../../utils/supportedChainId'
-const ArrowContainer = styled.div`
+
+export const ArrowContainer = styled.div`
   display: inline-block;
```

Comment:
Why does this component have display defined twice? I now it's not touched in this diff, but would be cool to clean up. Might be worth checking whether width and height 100% are required to be present as well, b/c that looks like a left-over.

Reply:
omg 😂